plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/build.properties

```diff
@@ -4,3 +4,4 @@ bin.includes = META-INF/,\
                .
 javacSource=1.7
 javacTarget=1.7
+additional.bundles = org.eclipse.emf.ecore
```

Comment:
Thanks, now things are a little clear to me. You should be able to remove `org.eclipse.emf.ecore` from MANIFEST.MF. It works for me.